### ISSUE TITLE
Need more questions regarding sale of the property 

### ISSUE THREAD

**Issue by tobyfey** [nonprofittechy/docassemble-MOHUDEvictionProject#113]
Did the new landlord give you notice 

Did that notice have a copy of the deed or a notarized document signed by both the previous owner and the new ones.

**Comment by tobyfey** [comment]
Ask Terry to check when done

**Comment by tobyfey** [comment]
Changed the interview question to:
id: sale of property
question: |
  Did your landlord ever sell the house while you were a tenant at this address?
fields:
  - Was your house sold to a new landlord while you lived in it?: previous_landlord_sold_property
    datatype: yesnomaybe
  - Did your new landlord give you a notice about the sale?: notice_of_sale_provided_new_landlord
    datatype: yesnomaybe
    show if: previous_landlord_sold_property
  - Did that notice have a copy of the deed, or a notarized document signed both by the original landlord and the new landlord?: notice_contains_sale_document
    datatype: yesnomaybe
    show if: notice_of_sale_provided_new_landlord

I got rid of notice from original landlord, since it wasn't mentioned in the defense

Output (changed landlord_tenant_name to landlord_doc_name)


The premises have been sold but the prior Landlord has not provided Tenant the
required notices and therefore no rent is owed
2. Prior owner of premises sold to Plaintiff. But Plaintiff did not provide Defendant the
required statutory notice of the sale. Therefore, no rent is owed per RSMo. § 535.081.
3. I have set forth reasons for dismissal of Plaintiff’s action against me, but other reasons
may also exist.
